roles/openshift_logging/defaults/main.yml

```diff
@@ -37,6 +37,9 @@ es_pvc_prefix: logging-es
 es_recover_after_time: 5m
 es_storage_group: 65534
 
+# allow cluster-admin or cluster-reader to view operations index
+es_ops_allow_cluster_reader: False
```

Comment:
not necessary, this is configured in the ES configuration (unless we want to update that value with this instead)